src/test/java/BlockDataExtractorTest.java

```diff
@@ -0,0 +1,35 @@
+import de.internetsicherheit.brl.bloxberg.cache.gui.BlockDataExtractor;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class BlockDataExtractorTest {
+
+    @DisplayName("does the json-File contain a timestamp? Requires at least 1 transaction.")
+    @Test
+    void testJsonFile() throws IOException {
+
+        // set up client
+        String[] args = new String[5];
+        args[0] = "https://core.bloxberg.org";
+        args[1] = "timestamps_test";
+        args[2] = "0";
+        args[3] = "1000";
```

Comment:
```suggestion
        args[3] = "1";
```

Reply:
done.
start is 33
stop is 34

Reply:
reason is: right now i only write out those timestamps that can be associated to a transaction. timestamps from empty blocks are ignored.